dev/tests/integration/framework/Magento/TestFramework/ErrorLog/Logger.php

```diff
@@ -50,15 +59,18 @@ public function getMessages()
     }
 
     /**
-     * @{inheritDoc}
+     * @inheritdoc
      *
-     * @param  integer $level   The logging level
+     * @param  int $level   The logging level
      * @param  string  $message The log message
      * @param  array   $context The log context
-     * @return Boolean Whether the record has been processed
+     * @return bool Whether the record has been processed
      */
-    public function addRecord($level, $message, array $context = [])
-    {
+    public function addRecord(
+        int $level,
+        string $message,
+        array $context = []
+    ): bool {
```

Comment:
Let's not spend space vertically, when it's not needed. Before it looked much better.
```suggestion
    public function addRecord(int $level, string $message, array $context = []): bool
    {
```